assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Footer.js

```diff
@@ -47,112 +47,103 @@ import SourceLink from '../../../../../components/SourceLink';
 import Data from 'googlesitekit-data';
 const { useSelect } = Data;
 
-const Footer = ( { metrics, selectedStats } ) => {
-	const propertyID = useSelect( ( select ) =>
-		select( MODULES_SEARCH_CONSOLE ).getPropertyID()
-	);
-	const isDomainProperty = useSelect( ( select ) =>
-		select( MODULES_SEARCH_CONSOLE ).isDomainProperty()
-	);
-	const referenceSiteURL = useSelect( ( select ) =>
-		untrailingslashit( select( CORE_SITE ).getReferenceSiteURL() )
-	);
-	const url = useSelect( ( select ) =>
-		select( CORE_SITE ).getCurrentEntityURL()
-	);
-	const dateRangeDates = useSelect( ( select ) =>
-		select( CORE_USER ).getDateRangeDates( {
-			offsetDays: DATE_RANGE_OFFSET,
-		} )
-	);
+function SourceLinkAnalytics( { id } ) {
+	const serviceURL = useSelect(
+		( select ) => {
+			const { getServiceReportURL } = select( MODULES_ANALYTICS );
+			const url = select( CORE_SITE ).getCurrentEntityURL();
+			const rangeDates = select( CORE_USER ).getDateRangeDates( {
```

Comment:
This may appear to be against our conventions for `useSelect`, but it's not – it returns a single value. It selects everything it needs internally to return that value which should make for less work.